eitprocessing/binreader/reader.py

```diff
@@ -7,6 +7,7 @@
 
 import io
 import struct
+import warnings
```

Comment:
I think a endian argument should result in an exception rather than a warning.
```python
```

Reply:
I went for the warning, because even if the endian is wrong and it is treated as non specified, the system default can be used which is in most cases correct. But the user has to be aware that there is mistake. I am also fine with the exception